core_backend/tests/api/test_manage_content.py

```diff
@@ -160,6 +160,71 @@ async def test_delete_content(
         assert response.status_code == 200
 
 
+class TestMultUserManageContent:
+    @pytest.mark.parametrize(
+        "content_title, content_text, content_metadata",
+        [("title 3", "test content 3", {})],
+    )
```

Comment:
parametrize makes sense if we want to test multiple different combos of parameters, but
seems like we're just using one! Fine to keep it though I think